src/js/actions/zones.js

```diff
@@ -32,14 +32,14 @@ export function asyncZoneDelete(zoneId) {
         dispatch(zoneDelete(zoneId));
 
         zoneDeleteZone(zoneId, function(error, response){
-            if(v4ResponseOk(response)) {
+            if(response) {
                 dispatch(zoneDeleteSuccess());
                 dispatch(dnsRecordClearAll(zoneId));
                 //after we provision a cname refresh the zone list
                 dispatch(asyncFetchZones());
             } else {
                 dispatch(zoneDeleteError());
-                dispatch(notificationAddError(response));
+                dispatch(notificationAddError(error.body.errors[0].message));
```

Comment:
Good catch.  This should be:
```
notificationAddClientAPIError(zoneDeleteError(),error.body.errors[0].message);
```
its long because the API supports multiple error messages even though we only get back one.